pkg/operation/shoot/shoot.go

```diff
@@ -357,14 +363,14 @@ func ComputeTechnicalID(projectName string, shoot *gardenv1beta1.Shoot) string {
 // ConstructInternalClusterDomain constructs the domain pointing to the kube-apiserver of a Shoot cluster
```

Comment:
~// ConstructInternalClusterDomain constructs the domain pointing to the kube-apiserver of a Shoot cluster~